### PR TITLE
Small edits and rearranged order for clarity

### PR DESCRIPTION
**Pull request by willemneal**

### CODE REVIEW

**Comment by jtenner** [comment]
Just need to make sure markdown lint passes.

Are there any markdown tools for grammar?

**Comment by jtenner** [comment]
```
README.md: 17: MD022/blanks-around-headings/blanks-around-headers Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below] [Context: "# Table of contents"]
README.md: 17: MD025/single-title/single-h1 Multiple top level headings in the same document [Context: "# Table of contents"]
README.md: 18: MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "1. [Usage](#usage)"]
README.md: 33: MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
README.md: 363: MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
README.md: 759: MD022/blanks-around-headings/blanks-around-headers Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Above] [Context: "## Using as-pect as a Package"]
```

These are the outstanding fixes that need to be made.

**Comment by jtenner** [comment]
```
Currently as-pect will compile each file that matches the glob's in the include property of your configuration. The default include is "assembly/__tests__/**/*.spec.ts".
```

I like the link here, but `glob's` does not possess something, so it is grammatically incorrect to use the apostrophe here. 

Please remove the `'` in `glob's`.

File: README.md [hunk 1]
```diff
@@ -240,7 +211,17 @@ run the `asp` command line tool to use the `SummaryTestReporter` like this:
...
 ```
 
-## Compiler Flags
+## AssemblyScript Compiler Options
+
+### `--path` CLI Argument
+
+By default `as-pect` will use node's resolver to look for an `assemblyscript`
+module.  If you want to specify a different version of the compiler, uses
+`--path ../relative/path/to/compiler/folder`.  Note that it expects the following
```
[the remark this answers is not in the document]
Comment (willemneal) [reply README.md]: Weird it is compiler in the cli options...

**Pull request merged**